src/main/java/org/broadinstitute/hellbender/tools/exome/CreatePanelOfNormals.java

```diff
@@ -279,92 +278,54 @@ protected void runPipeline(JavaSparkContext ctx) {
-        }
-        return targetFactorThreshold;
-    }
-
-    /**
-     * Checks that the user input {@link #outlierTruncatePercentileThresh} is correct.
-     * @return the user input {@link #outlierTruncatePercentileThresh}.
-     */
-    private double checkCountTruncatePercentile() {
-        if (outlierTruncatePercentileThresh < 0 || outlierTruncatePercentileThresh > 50.0 || Double.isNaN(outlierTruncatePercentileThresh)) {
-            throw new UserException.BadArgumentValue(COUNT_TRUNCATE_PERCENTILE_FULL_NAME, "the value must be in the [0, 50.0) range");
-        }
-        return outlierTruncatePercentileThresh;
-    }
-
-    private double checkTargetMaximumZeroPercentage() {
-        if (maximumPercentZerosInTarget < 0 || maximumPercentZerosInTarget > 100 || Double.isNaN(maximumPercentZerosInTarget)) {
-            throw new UserException.BadArgumentValue(MAXIMUM_PERCENT_ZEROS_IN_TARGET_FULL_NAME, "the value must be in the [0, 100.0) range");
-        }
-        return maximumPercentZerosInTarget;
-    }
-
-    private double checkColumnMaximumZeroPercentage() {
-        if (maximumPercentZerosInColumn < 0 || maximumPercentZerosInColumn > 100 || Double.isNaN(maximumPercentZerosInColumn)) {
-            throw new UserException.BadArgumentValue(MAXIMUM_PERCENT_ZEROS_IN_COLUMN_FULL_NAME, "the value must be in the [0, 100.0) range");
-        }
-        return maximumPercentZerosInColumn;
-    }
-
     /**
-     * Write a target weight file that can be read in by the R CBS.
-     *
-     * @param pon never {@code null}
-     * @param outputFile never {@code null}.  Must be writable.
-     */
-    public static void writeTargetWeightsFile(final PoN pon, final File outputFile) {
-        final double[] targetWeights = HDF5PoNCreator.calculateTargetWeights(pon);
-        ParamUtils.writeValuesToFile(targetWeights, outputFile);
-    }
-
-    /**
-     * Write a target weight file that can be read in by the R CBS.
-     *
-     * @param ponFile never {@code null}.  Must be readable.
-     * @param outputFile never {@code null}.  Must be writable.
+     * Read target variances from an HDF5 PoN file and write the corresponding target weights
+     * to a file that can be read in by R CBS.
+     * @param ponFile       never {@code null}, HDF5 PoN file
+     * @param outputFile    never {@code null}, output file
      */
-    public static void writeTargetWeightsFile(final File ponFile, final File outputFile) {
+    private static void writeTargetWeightsFile(final File ponFile, final File outputFile) {
         Utils.regularReadableUserFile(ponFile);
         try (final HDF5File file = new HDF5File(ponFile, HDF5File.OpenMode.READ_ONLY)) {
-            final HDF5PoN pon = new HDF5PoN(file);
-            writeTargetWeightsFile(pon, outputFile);
+            final HDF5PCACoveragePoN pon = new HDF5PCACoveragePoN(file);
+            final double[] targetWeights = DoubleStream.of(pon.getTargetVariances()).boxed().mapToDouble(v -> 1/v).toArray();
```

Comment:
oops, good catch!  not sure if the public dependency has been updated yet in my branch, so I don't see `applyToArray`, will check again after a rebase